diff --git a/Configurations/15-ios.conf b/Configurations/15-ios.conf
--- a/Configurations/15-ios.conf	2021-12-14 15:45:01.000000000 +0000
+++ b/Configurations/15-ios.conf	2022-01-18 16:12:24.000000000 +0000
@@ -31,7 +31,18 @@
     "iossimulator-xcrun" => {
         inherit_from     => [ "ios-common" ],
         CC               => "xcrun -sdk iphonesimulator cc",
+		cflags           => add("-arch i386"),
     },
+	"iossimulator64-xcrun" => {
+		inherit_from     => [ "ios-common" ],
+		CC               => "xcrun -sdk iphonesimulator cc",
+		cflags           => add("-arch x86_64"),
+	},
+	"iossimulator-arm64-xcrun" => {
+		inherit_from     => [ "ios-common" ],
+		CC               => "xcrun -sdk iphonesimulator cc",
+		cflags           => add("-arch arm64"),
+	},
 # It takes three prior-set environment variables to make it work:
 #
 # CROSS_COMPILE=/where/toolchain/is/usr/bin/ [note ending slash]
